crds/operators.coreos.com_operatorconditions.yaml

```diff
@@ -0,0 +1,91 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.3.0
+  creationTimestamp: null
+  name: operatorconditions.operators.coreos.com
+spec:
+  group: operators.coreos.com
+  names:
+    kind: OperatorCondition
+    listKind: OperatorConditionList
+    plural: operatorconditions
+    singular: operatorcondition
+  scope: Namespaced
+  versions:
+  - name: v1
```

Comment:
Is this really a v1 API or should we keep it in line with the other OLM types as v1alpha1? v1 is effectively saying it will never change

Reply:
Until we have an upstream/downstream split new CRDs need to be introduced at v1, this is an openshift requirements born from the fact that we offer higher levels of support than guaranteed by lower api versions.

Reply:
Interesting -- Bit paradoxical that our oldest and most stable apis are at alpha and the newest apis are at v1 but I see the reasoning.